Remotion/Data/DomainObjects/Infrastructure/Serialization/FlattenedSerializationInfo.cs

```diff
@@ -64,7 +67,8 @@ private void AddFlattenedSerializable (IFlattenedSerializable? serializable)
     {
       if (serializable != null)
       {
-        var typeName = serializable.GetType().AssemblyQualifiedName;
+        var type = serializable.GetType()!;
```

Comment:
null-forgiving needed for c# to discover that the resulting typename is never null?

Reply:
I though it was because it was a result of the refactoring. But it seems like it is not needed, and that the nullability refactorings are error-prone and need re-checking....